cmd/rekor-cli/app/log_info.go

```diff
@@ -44,6 +44,7 @@ import (
 
 type logInfoCmdOutput struct {
 	TreeSize       int64
+	TotalTreeSize  int64
```

Comment:
Could you add some documentation either here or in the output that `Tree Size` is the active shard size and `Total Tree Size` is Active and Inactive Tree?

Maybe Tree Size -> Active Tree Size

Reply:
I think `ActiveTreeSize` would be a better name! I was worried about making a breaking change in case people are grabbing `TreeSize` via `jq` or something, but I think renaming it is definitely more clear.

Reply:
+1 on doing the breaking change now before 1.0 and putting it in the release notes!

Reply:
SGTM, added in the release note.